engine/src/main/java/org/terasology/world/WorldProvider.java

```diff
@@ -29,19 +31,49 @@
     /**
      * An active block is in a chunk that is available and fully generated.
      *
-     * @param pos
+     * @param pos The position
      * @return Whether the given block is active
+     * @deprecated This is scheduled for removal in an upcoming version
+     *             method will be replaced with JOML implementation {@link #isBlockRelevant(Vector3ic)}.
      */
+    @Deprecated
     boolean isBlockRelevant(Vector3i pos);
 
+    /**
+     *
```

Comment:
Some sort of descriptor for the method itself here please :-) Looks like the old method has one, can just copy it forward.

Reply:
```suggestion
     * An active block is in a chunk that is available and fully generated.
```